notebooks/official/migration/sdk-custom-image-classification-custom-container.ipynb

```diff
@@ -150,180 +159,137 @@
-        "If you are in a live tutorial session, you might be using a shared test account or project. To avoid name collisions between users on resources created, you create a uuid for each instance session, and append it onto the name of resources you create in this tutorial.\n"
+        "Authenticate your environment on Google Colab."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "timestamp"
+        "id": "46604f70e831"
       },
       "outputs": [],
       "source": [
-        "import random\n",
-        "import string\n",
-        "\n",
+        "import sys\n",
         "\n",
-        "# Generate a uuid of a specifed length(default=8)\n",
-        "def generate_uuid(length: int = 8) -> str:\n",
-        "    return \"\".join(random.choices(string.ascii_lowercase + string.digits, k=length))\n",
+        "if \"google.colab\" in sys.modules:\n",
         "\n",
+        "    from google.colab import auth\n",
         "\n",
-        "UUID = generate_uuid()"
+        "    auth.authenticate_user()"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "sBCra4QMA2wR"
+        "id": "91842ef41bbd"
       },
       "source": [
-        "### Authenticate your Google Cloud account\n",
-        "\n",
-        "To authenticate your Google Cloud account, follow the instructions for your Jupyter environment:\n",
-        "\n",
-        "**1. Vertex AI Workbench**\n",
-        "<br>You are already authenticated.\n",
+        "### Set Google Cloud project information\n",
         "\n",
-        "**2. Local JupyterLab instance**\n",
-        "<br>Uncomment and run the following code:"
+        "To get started using Vertex AI, you must have an existing Google Cloud project and [enable the Vertex AI API](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com). Learn more about [setting up a project and a development environment](https://cloud.google.com/vertex-ai/docs/start/cloud-environment)."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "254614fa0c46"
+        "id": "294fe4e5a671"
       },
       "outputs": [],
       "source": [
-        "# ! gcloud auth login"
+        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+        "LOCATION = \"us-central1\"  # @param {type:\"string\"}"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "ef21552ccea8"
+        "id": "0bd7a5e762cc"
       },
       "source": [
-        "**3. Colab**\n",
-        "<br>Uncomment and run the following code:"
+        "### UUID\n",
+        "If you're in a live tutorial session, you might be using a shared test account or project. To avoid name collisions between users on resources created, you create a uuid for each instance session, and append it onto the name of resources you create in this tutorial."
```

Comment:
remove second "you"
- To avoid name collisions between users on resources created, create a uuid for each instance session